routes/data-api/api-functions.js

```diff
@@ -11,6 +11,7 @@ var Activity = model.Activity;
 var IPCCActivity = model.IPCCActivity;
 var Data = model.Data;
 var app_logger = require('../common/logger');
+var widestage = require('../../services/widestage.js');
```

Comment:
@gilgad should widestage be an application dependency in the `package.json` file?

Reply:
No, Widestage is a standalone instance. That widestage file that is being imported exists in our app and manages communication between our app and the Widestage platform.